src/module.c

```diff
@@ -2462,6 +2470,14 @@ NAPI_METHOD(dcn_msg_set_quote) {
   NAPI_RETURN_UNDEFINED();
 }
 
+NAPI_METHOD(dcn_msg_remove_quote) {
```

Comment:
afail, most `dc_*` core-functions map 1:1 to `dcn_*` node functions.

unless there are good reasons to not do so, i'd stay with this approach and keep things consistent. so, remove `dcn_msg_remove_quote()` can use `dcn_msg_set_quote(NULL)` instead. that would also remove some lines of code.

Reply:
I spend like 2 hours trying to do this properly, unless you find a way to detect JS's null object we will stay with this workaround.
Or we could sometime rewrite the whole nodebindings properly with napi.rs, but not now.

Reply:
> I spend like 2 hours trying to do this properly, unless you find a way to detect JS's null object in c we will stay with this workaround.

i'd say that just the same as with `dcn_set_draft` that also passes a null-message-pointer from js to c.

i always do these kind of things like that, copy+paste a similar function :) eg. also for the java-bindings where i also tend to forget the details :)

but sure, as said, no blocker. if the current doubled function works and is tested, it's fine. but maybe sth. for the future.

Reply:
thanks for pointing that out,
```c
dc_msg_t* dc_msg_quote = NULL;
napi_get_value_external(env, argv[1], (void**)&dc_msg_quote);
```
seems to work, thanks 👍